static/js/libs/channels.js

```diff
@@ -258,4 +258,22 @@ function sortChannels(channels, options) {
   };
 }
 
-export { sortChannels, parseChannel, createChannelTree, sortAlphaNum };
+/**
+ * Get a channel string based on an object containing track, risk and branch
+ *
+ * @param {track: string, risk: string, branch: string} channelObj
+ * @returns string
+ */
+function getChannelString(channelObj) {
```

Comment:
Seems similar to `release/helpers.js@getChannelName`.

Reply:
I feel like `release/helpers.js@getChannelName` is quite specific for the release UI internals, with the AVAILABLE constant. This was meant to be a more generic builder that can build a string from either the object created by `libs/channels.js@parseChannel` or what the API returns. Thoughts?

Reply:
Ah, right, the `AVAILABLE` risk edge case. Would be good to get rid of this (now that we moved the 'available' out of the table to the tabs). But that's another significant refactor...

Maybe the `getChannelName` helper can use `getChannelString` internally for now, to have one place where we construct channel name string.